### PR TITLE
d2: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/d2.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+`d2 fmt {{ruta/al/archivo_de_entrada.d2}}`
+
+- Lista de temas disponibles:
+
+`d2 themes`
+
+- Usa un [t]ema diferente para el archivo de salida (enumera primero los temas disponibles para obtener el `theme_id` deseado):
+
+`d2 --theme {{identificador_tema}} {{ruta/al/archivo_de_entrada.d2}} {{ruta/al/archivo_de_salida.svg|ruta/al/archivo_de_salida.png}}`
```
Comment (tricantivu) [inline pages.es/common/d2.md:24]:
```suggestion
`d2 --theme {{identificador_tema}} {{ruta/al/archivo_de_entrada.d2}} {{ruta/al/archivo_de_salida.[svg|png]}}`
```

File: pages.es/common/d2.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Hace que los diagramas renderizados parezcan bocetos hechos a mano:
+
+`d2 --sketch true {{ruta/al/archivo_de_entrada.d2}} {{ruta/al/archivo_salida.svg|ruta/al/archivo_de_salida.png}}`
```
Comment (tricantivu) [inline pages.es/common/d2.md:28]:
```suggestion
`d2 --sketch true {{ruta/al/archivo_de_entrada.d2}} {{ruta/al/archivo_de_salida.[svg|png]}}`
```

File: pages.es/common/d2.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Formatea un archivo fuente D2:
+
+`d2 fmt {{ruta/al/archivo_de_entrada.d2}}`
+
+- Lista de temas disponibles:
+
+`d2 themes`
+
+- Usa un [t]ema diferente para el archivo de salida (enumera primero los temas disponibles para obtener el `theme_id` deseado):
```
Comment (tricantivu) [inline pages.es/common/d2.md:22]:
```suggestion
- Usa un [t]ema diferente para el archivo de salida (primero enumera los temas disponibles para obtener el `theme_id` deseado):
```

**Review by tricantivu — COMMENTED** [review]
I believe the English page placeholders for output files should be updated:

```markdown
{{path/to/output_file.[svg|png]}}
```

**Review by tricantivu — APPROVED** [review]

**Comment by kant** [comment]
@sebastiaanspeck Done

**Review by kbdharun — COMMENTED** [review]
@tricantivu We don't support the optional placeholder usage (since the style guide refresh a month ago), feel free to checkout https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#optional-placeholders. I will mark this pull request as draft for now and will update the English page PR for the suggested usage.

File: pages.es/common/d2.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# d2
+
+> Un lenguaje moderno de scripting de diagramas que convierte texto en diagramas.
+> Más información: <https://d2lang.com/tour/man>.
+
+- Compila y renderiza un archivo fuente D2 en un archivo SVG o PNG:
```
[the remark this answers is not in the document]
Comment (spageektti) [reply pages.es/common/d2.md:6]: The second line of description, which was recently added to the English page, is missing.
`> Note: the output file supports SVG and PNG file formats.`

Comment (spageektti) [inline pages.es/common/d2.md:6]: The description of the first example was recently updated from:
`- Compile and render a D2 source file into an SVG or PNG file:`
 to:
`- Compile and render a D2 source file into an output file:`.
This should be updated there too.

File: pages.es/common/d2.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# d2
+
+> Un lenguaje moderno de scripting de diagramas que convierte texto en diagramas.
+> Más información: <https://d2lang.com/tour/man>.
```
[the remark this answers is not in the document]
Comment (kant) [reply pages.es/common/d2.md:5]: done

[the remark this answers is not in the document]
Comment (kant) [reply pages.es/common/d2.md:5]: done

File: pages.es/common/d2.md (hunk 4 above)
Comment (kant) [reply pages.es/common/d2.md]: Done

**Review by tricantivu — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**